equinox-shipping/Domain/FinalizationProcessManager.fs

```diff
@@ -0,0 +1,54 @@
+module FinalizationProcessManager
+
+    type Service(transactions: FinalizationTransaction.Service, containers: Container.Service, shipments: Shipment.Service) =
+
+        // Transaction ID can be a md5 hash of all shipments ID
+        let execute (transactionId: string) : FinalizationTransaction.Events.Event option -> Async<bool> =
+            let rec loop (update: FinalizationTransaction.Events.Event option) =
+                async {
+                    let loop event = loop (Some event)
+
+                    let! action =
+                        transactions.Apply(transactionId, update)
+
+                    match action with
+                    | FinalizationTransaction.Action.AssignShipments (containerId, shipmentIds) ->
+                        let! result =
+                            seq {
+                                for sId in shipmentIds ->
+                                    async {
+                                        let! res = shipments.Execute(sId, Shipment.Command.Assign containerId)
```

Comment:
maybe move `async {` to end of previous line ?

Reply:
Moved thanks!